design/FY2024/NFP-labHeatExchangerHR.md

```diff
@@ -0,0 +1,278 @@
+
```

Comment:
I have no problem with increasing the accuracy of the HX model. The manufacturers contacted during this model's development said that the HX should not be operated outside the 50% - 130% air flow range, where the effectiveness is roughly linear. If there are curves available for better accuracy then that is a good thing. The curve above is shown for a 10C delta T. Does HX eff vary at other delta Ts or is air flow the driving factor? are there other delta T curves available? or equations of effectiveness that describe performance at other delta T? and/or detla w? where these curves could be created as bi-quadratic/bivariate.

Reply:
Currently, this is one example curve I saw. I'll search around for some other available curves. I will also contact John McHugh's team and see if they have more info.